src/tools/_utilities.tool.scss

```diff
@@ -9,7 +11,7 @@
 @mixin utility-styles($properties, $name, $value, $variable: null) {
   @each $property in $properties {
     $attributeString: "";
-    @if $variable != null {
+    @if $variable {
```

Comment:
Please double check if this is the desired logic.
Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-if-no-null/README.md